git-clang-format.py

```diff
@@ -89,7 +89,6 @@ def main():
       'js',  # JavaScript
       'ts',  # TypeScript
       'cs',  # C Sharp
-      'json',  # Json
```

Comment:
I use this to format the `cmake presets` and other json files!

Reply:
What clang-format version are you using? I tried it with clang-format 18.0.0 (`clang-format version 18.0.0 (/mnt/agent/work/llvm-project/llvm-project/clang 367a7b9b0f9ac6e2a97cdfda4d29436c8713c254 based on LLVM d374a78545770f0325eca45a672b5b881798e87d revision)`) and it didn't work with json enabled. 
Is there may a way to add new extensions but still be backward compatible?

Reply:
V17 and newer